subsys/bluetooth/audio/micp_mic_dev.c

```diff
@@ -25,11 +25,16 @@
 
 LOG_MODULE_REGISTER(bt_micp, CONFIG_BT_MICP_MIC_DEV_LOG_LEVEL);
 
+#define NOTIFY_RETRY_DELAY_MS(_retry_count)  (100 * (_retry_count))
+#define NOTIFY_RETRY_COUNT_MAX 3
```

Comment:
Why three?

Reply:
Overall this PR looks good to me. I'm just wondering why the max retry of three. It seems arbitrary.
If spec. requires that we deliver notifications, should we not continue forever, or until the peer has disconnected?